lldb/test/API/functionalities/data-formatter/data-formatter-stl/libcxx/span/TestDataFormatterLibcxxSpan.py

```diff
@@ -172,7 +172,4 @@ def test_ref_and_ptr(self):
 
         # The pointer should just show the right number of elements:
 
-        ptrAddr = self.findVariable("ptr").GetValue()
-        self.expect_expr(
-            "ptr", result_type="std::span<int, 5> *", result_summary=f"{ptrAddr} size=5"
-        )
+        self.expect("frame variable ptr", patterns=["ptr = 0x.*", " size=5"])
```

Comment:
`SBValue::GetValue` essentially uses the summary string provided by summary provider. So it returns "size=5" after I removed the pointer value in libcxx container summary.

Reply:
That makes sense, but you can still use `expect_expr` to match this. You just need to say `result_summary=re.compile("your_pattern")`